new/LIT5890AsmatGa.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5890AsmatGa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ʾAsmāt-prayer of the Archangel Gabriel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-04-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/><!---->
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit">
+                    <ab>
+                        ድርሳን፡ ዘቅዱስ፡ ገብርኤል፡ ዘይትነበብ፡ አመ፡ ፲ወ፱፡ ለታኅሣሥ፡ ጤድኅናኤል፡ ሕውታኤል፡ አውዳኤል፡ ተርቡታኤል፡ ቡኤል፡ ግዔል፡ ዝኤል፡ እሎንተ፡ ቃላተ፡ ዘነገሮ፡ እግዚአብሔር፡ ለቅዱስ፡ ገብርኤል፡
```

Comment:
just FYI, since it is a new feature: it is possible to mark up such dates with `<date ref="ethiocal:Tahsas19">`...
https://betamasaheft.eu/Guidelines/?id=dates
(as always: the possibility of encoding does not make it compulsory, you decide whether it's relevant for you or not)